### ISSUE TITLE
Myelomalacia

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#9733]
**Preferred term label:**
Myelomalacia

**Synonyms**

**Definition (free text, please give PubMed ID)**
Softening of the spinal cord with high intensity signal changes on T2-weighted images (PMID: 8010278; 34277695)

**Parent term (use hpo.jax.org/app)**
Abnormal spinal cord morphology HP:0002143

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson
